### PR TITLE
add "Explore" section on index

### PR DESCRIPTION
**Pull request by guzzisti**
For my use case it's fine to show a random short link on the index page when it's called without a shortlink:
image

As this may not be viable for all use-cases if added a corresponding setting which defaults to `false`.

### CODE REVIEW

**Comment by flokX** [comment]
I don't think that we need such a functionality in devShort since it's a URL shortener for developers and other people who want it as simple and slim as possible. I think in most cases the short urls are private and should only be accessed by people who know them. Not everyone should know the short urls or the user want to track the accesses to the different short urls precisely. This feature would enlarge the codebase but is not so useful for the users. Anyway, thanks for your contribution!

**Comment by guzzisti** [comment]
Thank you for your feedback.
I agree that this feature suffices a rather specific use case and may not be necessarily merged into the main repo to keep the code base as small as possible.

But theres one point in your statement 
Refering to this as privacy-related is ridiculous. "Security through obscurity" (as in "people doesn't know the name") is not a valid security- nor privacy-concept. If you wan't to keep things private, you need to set up some sort of authentication (user need to login, restrict allow clients by some kind of attributes, whatsoever).
If you put things into public, you should always act as if these things are accessible by anyone - even if users don't know a name in the first place. It should be rather easy to write a short script to iterate through short URLs and check if you are redirected or not. As theres no brute-force protection of any kind, this is only throttled by the power of your webserver.

Long story short: the feature itself may not be that usefull, but hiding publicy available things does not conduce to privacy.

**Comment by flokX** [comment]
Your totally right, maybe it's worth to add a note about this problematic to the wiki. I know that it's a bad idea to include confidential things in the links. I meant with privat that we just shouldn't serve everyone all links on a silver platter. But I am pleased about the lively discussion!

**Comment by guzzisti** [comment]
good idea, i started a wiki page for this topic: https://github.com/flokX/devShort/wiki/Pitfalls-of-URL-shortening

We could fill this up over time with some more info about what people should keep in mind when using URL shortening.
